warp_aretomo_pytom_scripts/convert_pytomTMstar_to_pytomGUIxml.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# run these command first to load required modules on angstrom
+# module load miniconda/pytom/latest
+# condaactivate
+
+#Loop converts star file generated by pytom_tm into a .xml file readible by the pytomGUI
+
+for i in *.mrc;
+do
+	cp "$i/tm_results/${i%.mrc}_particles.star" "$i/tm_results/${i%.mrc}_particles_rln.star"
+	sed -i "s/_ptm/_rln/" "$i/tm_results/${i%.mrc}_particles_rln.star"
+	sed -i "s/_rlnLCCmax/_rlnOriginXAngst/" "$i/tm_results/${i%.mrc}_particles_rln.star"
+	sed -i "s/_rlnCutOff/_rlnOriginYAngst/" "$i/tm_results/${i%.mrc}_particles_rln.star"
+	sed -i "s/_rlnSearchStd/_rlnOriginZAngst/" "$i/tm_results/${i%.mrc}_particles_rln.star"
+	sed -i "s/_rlnDetectorPixelSize/_rlnPixelSize/" "$i/tm_results/${i%.mrc}_particles_rln.star"
+	convert.py -f "$i/tm_results/${i%.mrc}_particles_rln.star" -o "xml"
+	mv "${i%.mrc}_particles_rln.xml" "$i/tm_results/."  

```

Comment:
Please rename the file to `*_ptm.star` to make sure this file is never read by relion as it has broken headers (`_rlnOriginXAngst`, `_rlnOriginYAngst`, `_rlnOriginZAngst`)

```suggestion
	cp "$i/tm_results/${i%.mrc}_particles.star" "$i/tm_results/${i%.mrc}_particles_ptm.star"
	sed -i "s/_ptm/_rln/" "$i/tm_results/${i%.mrc}_particles_ptm.star"
	sed -i "s/_rlnLCCmax/_rlnOriginXAngst/" "$i/tm_results/${i%.mrc}_particles_ptm.star"
	sed -i "s/_rlnCutOff/_rlnOriginYAngst/" "$i/tm_results/${i%.mrc}_particles_ptm.star"
	sed -i "s/_rlnSearchStd/_rlnOriginZAngst/" "$i/tm_results/${i%.mrc}_particles_ptm.star"
	sed -i "s/_rlnDetectorPixelSize/_rlnPixelSize/" "$i/tm_results/${i%.mrc}_particles_ptm.star"
	convert.py -f "$i/tm_results/${i%.mrc}_particles_ptm.star" -o "xml"
	mv "${i%.mrc}_particles_ptm.xml" "$i/tm_results/."  

```